mlrun/config.py

```diff
@@ -240,6 +240,13 @@
             "channel": "master",
         },
     },
+    "storage": {
+        # What type of auto-mount to use for functions. Can be one of: none, auto, v3io_cred, v3io_fuse, pvc.
+        # Default is auto - which is v3io_cred when running on Iguazio and none otherwise
```

Comment:
Also, why wouldn't auto be `v3io_cred` when running on Iguazio, `pvc` when pvc is configured, none otherwise (similar to `auto_mount`) ?

Reply:
There is no point in doing complex logic here - if the user wants to do something fancy, he can do it by using the sdk for the functions. The idea here is to be very clear with what the default mount policy should be. If you want `pvc`, then configure it. The only reason `auto` points at v3io-credentials when running on Iguazio is because it's a relatively painless action with no resource drain on anything, and it's what you want to do in 99% of the cases when you run Iguazio.